taichi/runtime/metal/shaders/runtime_structs.metal.h

```diff
@@ -114,13 +114,13 @@ STR(
 
     struct SNodeExtractors {
       struct Extractor {
-        int32_t start = 0;
         int32_t num_bits = 0;
         int32_t acc_offset = 0;
-        int32_t num_elements_from_root = 0;
+        int32_t shape = 0;
+        int32_t acc_shape = 0;
       };
-
       Extractor extractors[kTaichiMaxNumIndices];
+      bool packed;
```

Comment:
better assign an initial value to `packed` since there's no well-designed constructor

Reply:
Sure. Let me merge first to unblock branch cut and put this along with a future PR.